CMakeLists.txt

```diff
@@ -157,9 +154,19 @@ add_subdirectory(src)
 if (ENABLE_TESTS)
   add_subdirectory(tests)
 endif ()
-if (ENABLE_BENCHMARKS)
+
+if (ENABLE_DEVELOPER_BENCHMARKS)
```

Comment:
Question: What happens if `ENABLE_DEVELOPER_BENCHMARKS=On` and `ENABLE_BENCHMARKS=Off`?

Reply:
Enable_benchmarks turns on blt GoogleBenchmark support, and our enable_developer_benchmarks turns on our Umpire benchmarks. So if the developer_benchmarks is on and the enable_benchmarks is off, then only the stress tests will be run. If they are both on, then both stress tests and google benchmarks will be run. If enable_benchmarks is on, but the developer_benchmarks is off, then it will only enable GoogleBenchmark in BLT and I have that warning statement printed out that no actual benchmarks will be built.